app/src/common/utils/connectRouter.jsx

```diff
@@ -47,7 +56,7 @@ export const connectRouter = (mapURLParamsToProps = takeAll, queryUpdaters = {},
       return mappedUpdaters;
     },
     (stateProps, dispatchProps, ownProps) => ({
-      ...omit(stateProps, ['namespace', 'namespacedQuery', 'namespaceSelector']),
+      ...omit(stateProps, ['namespacedQuery', 'namespaceSelector']),
```

Comment:
If you need namespace, it's better to get it in the corresponding component using `connect`. These props are not this component's interface, that's why they are omitted here.

Reply:
Fixed.

Reply:
Fixed.